src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -379,6 +379,8 @@ private void ProcessDownloadedBlock(ChainedHeaderBlock chainedHeaderBlock)
                     chainedHeaderBlocksToValidate = this.chainedHeaderTree.PartialValidationSucceeded(chainedHeaderBlock.ChainedHeader, out fullValidationRequired);
                 }
 
+                this.logger.LogTrace("Full validation required: {0}.", fullValidationRequired);
```

Comment:
quite ugly in log file, i prefer to use something like
```
 this.logger.LogTrace("Full validation is{0} required.", fullValidationRequired ? "" : " NOT");
```

Reply:
Fixed